docker/conf/php.ini

```diff
@@ -0,0 +1,8 @@
+; PHP.ini required and recommended options.
+; Used by GitLab-CI and docker-compose
+
+[PHP]
+memory_limit = -1
+
+[Date]
+date.timezone = Europe/Paris
```

Comment:
Well, I set this to not get PHP yelling.

But this can indeed be great if it can be configurable.

Any suggestion?